doc/man7/OSSL_PROVIDER-default.pod

```diff
@@ -193,6 +193,8 @@ The OpenSSL default provider supports these operations and algorithms:
 =over 4
 
 =item RSA, see L<EVP_KEM-RSA(7)>
+=item X25519, see L<EVP_KEM-X25519(7)>
+=item EC, see L<EVP_KEM-EC(7)>
```

Comment:
You *must* have empty lines separating the `=item` lines.  I know it's annoying, but the POD reader strictly reads in paragraph mode, so without the empty lines, what you get here is one item, looking like this:

```
=item RSA, see L<EVP_KEM-RSA(7)> =item X25519, see L<EVP_KEM-X25519(7)> =item EC, see L<EVP_KEM-EC(7)>
```

Reply:
Thanks, I forget that sometimes..